packages/patternfly-4/react-core/src/components/SimpleList/SimpleListGroup.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/SimpleList/simple-list';
+
+export interface SimpleListGroupProps {
+  /** Content rendered inside the SimpleList group */
+  children?: React.ReactNode;
+  /** Additional classes added to the SimpleList <ul> */
+  className?: string;
+  /** Additional classes added to the SimpleList group title */
+  titleClassName?: string;
+  /** Title of the SimpleList group */
+  title?: string | React.ReactNode;
```

Comment:
I believe ReactNode can be string already so not really needed to specify string explicitly